pom.xml

```diff
@@ -95,13 +95,18 @@ under the License.
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<hadoop.version>2.8.5</hadoop.version>
+		<flink.XmxITCase>2048m</flink.XmxITCase>
+		<flink.XmxUnitTest>1024m</flink.XmxUnitTest>
```

Comment:
I would be in favor of just hard-coding these; I don't see a need to make these configurable; you'd rather want to tweak the number of forks.

Reply:
My intention here was to have the memory and forkCount defined in one place, close to one another, as they are very tightly connected, to make this connection more obvious and easier to spot. So I would prefer to keep it as it is here. Also this way we reduce a chance that someone will copy out those hardcoded values somewhere else without thinking everything through introducing some subtle intermittent problems with build instability.